app/views/prescriptions/confirm.html.erb

```diff
@@ -53,41 +53,122 @@
+            </div>
+          </div>
+        <% end %>
+
+        <% unless @indicator_prescription.remaining_deductible_amt.blank? %>
+          <div class='row'>
+            <div class="col-md-12">
+              <b>Deductible Amount Remaining</b><br>
+              <%= "$#{@indicator_prescription.remaining_deductible_amt}" %><br><br>
+            </div>
+          </div>
+        <% end %>
+
+        <% if @substituted_drug %>
+          <div class='row'>
+            <div class="col-md-12">
+              <span class="badge">Substitution</span><%= " #{ @indicator_prescription.drug_substitution_help_message}" %>
+            </div>
+          </div>
+        <% end %>
+
+        <% unless @indicator_prescription.additional_content.blank? %>
+          <div class='row'>
+            <div class="col-md-12">
+              <% @indicator_prescription.additional_content.each do |content| %>
+                <%= content['text'] %><br>
+                <% unless content['href'].blank? %>
+                   <%= link_to content['href'], content['href'], method: content['method'] %><br>
+                <% end %>
+              <% end %>
+            </div>
+          </div>
+        <% end %>
+
         <div class='row'>
           <div class="col-md-12">
-            <small><%= @indicator_result.indicator.prescription.disclaimer_message %></small>
+            <small><%= @indicator_prescription.disclaimer_message %></small>
           </div>
         </div>
 
         <hr />
 
+        <div class='row'>
+          <% unless @pharmacy.name.nil? %>
+            <div class="form-group col-md-6">
+              <b>Preferred Pharmacy</b><br><br>
+                <%= @pharmacy.name %><br>
+                <%= @pharmacy.street %><br>
+                <%= "#{@pharmacy.city}, #{@pharmacy.state} #{@pharmacy.zip}" %><br>
+                <%= "P: #{format_phone(@pharmacy.phone)}" %><br>
+                <%= "F: #{format_phone(@pharmacy.fax)}" %>
+            </div>
+          <% end %>
+
+          <div class="form-group col-md-6">
+            <b>Prescribing Provider</b><br><br>
+              <% prescriber_info = prescriber_info(@patient.to_prescriber_hash) %>
+              <%= "#{prescriber_info.first_name} #{prescriber_info.last_name}"%><br>
+              <%= prescriber_info.address.street_1 %><br>
+              <%= address_info(prescriber_info.address) %><br>
+              <%= "P: #{format_phone(prescriber_info.phone_number)}" %><br>
+              <%= "F: #{format_phone(prescriber_info.fax_number)}" %>
+          </div>
+        </div>
+
         <div class='checkbox'>
           <%= label_tag :start_pa do %>
-            <% if @indicator_result.indicator.prescription.autostart %>
+            <% if @indicator_prescription.autostart %>
               <%= hidden_field_tag :start_pa, 1 %>
               <%= check_box_tag :start_pa, '1', true, disabled: true %>
               <b>Start PA?</b>
```

Comment:
Personally I like ```if something.present?``` instead of ```unless something.blank?``` but I think it was already done way two and I realize this is nitpicky so feel free to ignore